core/src/banking_stage/latest_unprocessed_votes.rs

```diff
@@ -115,9 +126,6 @@ impl LatestValidatorVotePacket {
     }
 }
 
-// TODO: replace this with rand::seq::index::sample_weighted once we can update rand to 0.8+
```

Comment:
Removing this comment because it is no longer accurate, we cannot use `sample_weighted` (see https://github.com/anza-xyz/agave/pull/2072#issuecomment-2224863047)